XRTK-Core/Packages/com.xrtk.core/Editor/Profiles/InputSystem/Controllers/Hands/BaseMixedRealityHandControllerDataProviderProfileInspector.cs

```diff
@@ -39,18 +41,27 @@ public override void OnInspectorGUI()
             if (showHandTrackingSettings)
             {
                 EditorGUI.indentLevel++;
+
                 EditorGUILayout.LabelField("Hand Rendering Settings");
                 EditorGUI.indentLevel++;
                 EditorGUILayout.PropertyField(handMeshingEnabled);
-                EditorGUI.indentLevel--;
                 EditorGUILayout.Space();
+                EditorGUI.indentLevel--;
+
                 EditorGUILayout.LabelField("Hand Physics Settings");
                 EditorGUI.indentLevel++;
                 EditorGUILayout.PropertyField(handPhysicsEnabled);
                 EditorGUILayout.PropertyField(useTriggers);
                 EditorGUILayout.PropertyField(boundsMode);
                 EditorGUILayout.Space();
                 EditorGUI.indentLevel--;
+
+                EditorGUILayout.LabelField("Tracked Hand Poses");
+                EditorGUI.indentLevel++;
+                EditorGUILayout.PropertyField(trackedPoses, true);
```

Comment:
This won't work correctly:

![image](https://user-images.githubusercontent.com/13334553/80807683-d34d6080-8b8b-11ea-96f0-c1d74a7165b4.png)

You'll have to manually loop through the array and display each property field by hand

Reply:
Actually this used to be a list, we should revert to that.

Reply:
Tracked poses still not rendering correctly in the inspectors

Reply:
Any update here?

Reply:
I have yet to see this issue appear for me. But instead of wasting anymore time here, probably going to revert to that ugly reordable list and wait for UIElements.